tests/framework/extensions/secrets/basicAuth.go

```diff
@@ -0,0 +1,24 @@
+package secrets
+
+import (
+	coreV1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func NewBasicAuthSecret(name, namespace, username, password string) *coreV1.Secret {
+	return &coreV1.Secret{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
This is redundant, the k8s client will derive this from the struct type.

Reply:
I don't understand what you mean. There are different types of secrets, my thought process was make that easier to create for the developer.